src/lib/localStorage.js

```diff
@@ -0,0 +1,11 @@
+export const jwtDecode = (t) => {
+  let token = {};
+  token.raw = t;
+  const tokenSplit = t.split(".");
+  const payloadBase64Url = tokenSplit[1];
```

Comment:
You should check the length of `tokenSplit` to ensure that `tokenSplit[1]` is not undefined.